packages/react-dom/src/__tests__/ReactDOMTextComponent-test.js

```diff
@@ -10,94 +10,99 @@
 'use strict';
 
 let React;
-let ReactDOM;
+let ReactDOMClient;
 let ReactDOMServer;
-
-// In standard React, TextComponent keeps track of different Text templates
-// using comments. However, in React Fiber, those comments are not outputted due
```

Comment:
I don't think we do this anymore. Test results didn't change when I removed this utility.